src/lib/mongoBackend/mongoNotifyContextAvailability.cpp

```diff
@@ -33,10 +33,16 @@
 
 /* ****************************************************************************
 *
-* mongoNofityContextAvailability -
+* mongoNotifyContextAvailability -
 */
-HttpStatusCode mongoNotifyContextAvailability(NotifyContextAvailabilityRequest* requestP, NotifyContextAvailabilityResponse* responseP, const std::string& tenant) {
-
+HttpStatusCode mongoNotifyContextAvailability
```

Comment:
Part of notifications is definitely supported, however, to really support it, func tests must be constructed and it must be tested.
I don't think it will be very hard to make subscriptions/notifications work with servicePath, but it is out of scope for this issue/PR.

NTC